server/fleet/vpp.go

```diff
@@ -138,3 +143,12 @@ const (
 	DefaultVPPInstallVerifyTimeout = 10 * time.Minute
 	DefaultVPPVerifyRequestDelay   = 5 * time.Second
 )
+
+type VPPAppUpdate struct {
```

Comment:
THANK YOU FOR THIS 🎉 🙏 

Do you mind renaming to something like `AppStoreAppUpdatePayload` or something along those lines? That way we start moving towards the "app store app" platform agnostic type names and concepts.